appdatabase/migrations/sql/0013_dont_disable_waku.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE settings DROP COLUMN waku_enabled;
```

Comment:
There's no drop column in sqlite, I know, it's weird :) You can leave this migration empty